client/landing/index.html

```diff
@@ -0,0 +1,161 @@
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+    <!-- Intro Header -->
+    <header class="intro">
+        <div class="intro-body">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8 col-md-offset-2">
+                        <h2>Memes For Mankind</h2>
+                        <p class="intro-text">One Chromecast, a few friends, and <br>a lifetime of awkward memories.</p>
+                        <a href="#about" class="btn btn-circle page-scroll">
+                            <i class="fa fa-angle-double-down animated"></i>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <!-- About Section -->
+    <section id="about" class="container content-section text-center">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2">
+                <h2>Memes For Life!</h2>
+                <p>Memes For Mankind is a party game.</p>
+                <p>Get 3+ friends to play and make sure they have the Google cast extension installed (laptop) or the mobile app (phone).</p>
+                <p>Start the game; a judge will be selected.<br>The judge then chooses a prompt for other players to creatively interpret.</p>
+                <p>After the judge then chooses his or her favorite submission, the winning entry is displayed on all devices.</p> <p>Share the meme on Facebook or Twitter.</p>
+                <p>Repeat until laughing subsides.</p>
+            </div>
+        </div>
+    </section>
+
+    <!-- Download Section -->
+    <section id="download" class="content-section text-center">
+        <div class="download-section">
+            <div class="container">
+                <div class="col-lg-8 col-lg-offset-2">
+                    <h2>Don't Forget The Cast Extension!</h2>
+                    <p>You'll need Chrome, a Chromecast, and the Google Cast Extension to play.</p>
+                    <a href="https://chrome.google.com/webstore/detail/google-cast/boadgeojelhgndaghljhdicfkmllpafd?hl=en" target="_blank" class="btn btn-default btn-lg">Get the Google Cast Extension</a>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Contact Section -->
+    <section id="contact" class="container content-section text-center">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2">
+                <h2>Get In Touch With Memes For Mankind</h2>
+                <p>Feel free to email us with praise, hate, love, and always memes!</p>
+                <p><a href="mailto:feedback@startbootstrap.com">feedback@memesformankind.com</a>
```

Comment:
The href="" needs to be updated here... still going to the bootstrap domain... not sure where ours should go... but not to bootstrap :)